extensions/ql-vscode/src/view/model-editor/LibraryRow.tsx

```diff
@@ -228,16 +232,18 @@ export const LibraryRow = ({
             methods={methods}
             modeledMethodsMap={modeledMethodsMap}
             modifiedSignatures={modifiedSignatures}
+            selectedSignatures={selectedSignatures}
             inProgressMethods={inProgressMethods}
             viewState={viewState}
             hideModeledMethods={hideModeledMethods}
             revealedMethodSignature={revealedMethodSignature}
             onChange={onChange}
+            onMethodClick={onMethodClick}
           />
           <SectionDivider />
           <ButtonsContainer>
             <VSCodeButton onClick={handleSave} disabled={!hasUnsavedChanges}>
```

Comment:
I spotted one odd piece of UI behaviour. If you have unsaved changes then the button will be enabled, but it could be that you haven't selected methods with unsaved changes, so pressing the button does nothing. It could be better if the button only became enabled when there were unsaved changes in the selected methods.

However I suggest we can leave this as a followup, since I believe we're expecting to do a design pass on this at some point anyway.

Reply:
Thanks for the review! That's a good point 🦅 👀 

But I also agree that it can be a follow-up change instead, along with any other design/user feedback!